### PR TITLE
Add anchor support to hittest sample

### PR DESCRIPTION
**Pull request by lehoangphu**
Anchor are mostly leveraged together with the hit-test. Thus, we should update the hittest sample to demonstrate the usage of Anchor and Hit-test together.

### CODE REVIEW

**Review by toji — COMMENTED** [review]
This is great, thank you!

I do have one request, though: You are correct that hit-tests are most useful with anchors, but since they CAN be used separately I'd like to keep the existing test the way it is to demonstrate "bare bones" hit test usage. If this test could be renamed to `hit-test-anchor.html` and put in the test list next to the original hit test sample, it would serve as a useful resource for developers to compare the two and better understand how anchors improve this use case. (It will also offer developers a simple way to A/B test a scenario with anchors and without, which will hopefully demonstrate why they're useful.)

**Comment by lehoangphu** [comment]
Agree.  I put back hit-test.html and keep it together with the new hit-test-anchors.html

File: hit-test-anchors.html [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+  <head>
+    <meta charset='utf-8'>
+    <meta name='viewport' content='width=device-width, initial-scale=1, user-scalable=no'>
+    <meta name='mobile-web-app-capable' content='yes'>
+    <meta name='apple-mobile-web-app-capable' content='yes'>
+    <link rel='icon' type='image/png' sizes='32x32' href='favicon-32x32.png'>
+    <link rel='icon' type='image/png' sizes='96x96' href='favicon-96x96.png'>
+    <link rel='stylesheet' href='css/common.css'>
+
+    <title>Hit Test</title>
```
Comment (toji) [inline hit-test-anchors.html:32]:
```suggestion
    <title>Hit Test with Anchors</title>

```

File: hit-test-anchors.html [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+  </head>
+  <body>
+    <header>
+      <details open>
+        <summary>Hit Test</summary>
```
Comment (toji) [inline hit-test-anchors.html:37]:
```suggestion
        <summary>Hit Test with Anchors</summary>

```

File: hit-test-anchors.html [hunk 3]
```diff
@@ -0,0 +1,236 @@
+<!doctype html>
+<!--
+Copyright 2018 The Immersive Web Community Group
```
Comment (toji) [inline hit-test-anchors.html:3]:
```suggestion
Copyright 2021 The Immersive Web Community Group

```

File: hit-test-anchors.html [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+    <link rel='stylesheet' href='css/common.css'>
+
+    <title>Hit Test</title>
+  </head>
+  <body>
+    <header>
+      <details open>
+        <summary>Hit Test</summary>
+        <p>
+          This sample demonstrates use of hit testing to place virtual objects on real-world surfaces.
```
Comment (toji) [inline hit-test-anchors.html:39]: Please update this description to indicate how this sample uses anchors.

**Review by toji — CHANGES_REQUESTED** [review]
Thank you! I've requested a few minor, non-code updates but once that's been done I'm happy to land this.

**Comment by lehoangphu** [comment]
Thank you for the feedbacks.  I have updated the title and description to better reflect the change.

**Review by toji — APPROVED** [review]

**Comment by toji** [comment]
Thank you!

**Pull request merged**